scheduler-interface-spec.md

```diff
@@ -910,27 +775,3 @@ message EventRecord {
 }
 ```
 
-### Auto Scaling Metrics
-
-Auto scaling metrics can be monitored and collected by a 3rd party auto-scaler and used as the
-supplementary metrics while making auto-scaling decisions. These metrics are indicating how many
-outstanding resource requests are there in the cluster, due to the shortage of node resources.
-This has excluded requests could not be satisfied because of other reasons, such as hitting queue
-resource limit, user/app limit, desire on certain placement constraints, etc.
-
-```protobuf
-// auto scaling metrics at a certain point of time
-message AutoScalingMetrics {
-  // a list of outstanding requests that desires for additional resources
-  repeated OutstandingResourceRequest outstandingRequests = 1;
-}
```

Comment:
I am not sure about this part. @yangwwei I see that you merged this change a few months ago from the autoscaling branch. Is this part of the autoscaling feature started a while ago, but not finished? If yes, I think we can remove it and we will add it back when we will really need it.

Reply:
hi @kingamarton  I think we can remove it as it is currently not used by any service.
We can always track back it won't be a problem when we want to resume the work on this in the future.